utils/_context/_scenarios/__init__.py

```diff
@@ -953,6 +953,17 @@ class _Scenarios:
         extproc_volumes={"./tests/appsec/blocking_rule.json": {"bind": "/appsec_blocking_rule.json", "mode": "ro"}},
     )
 
+    appsec_collect_all_headers = EndToEndScenario(
```

Comment:
Is there an existing scenario that cuold host those tests ? (an exsiting appsec scenarion, or one of confif_non_default)? 

Scenario are the bottleneck of system-tests 😓

Reply:
Not really, is new feature with new "DD_APPSEC_COLLECT_ALL_HEADERS" and "DD_APPSEC_HEADER_COLLECTION_REDACTION_ENABLED" config variables :S

Reply:
but we could probably merge it with one of the config_non_default, probably one of those aren't relying on old header collection mecanics

Reply:
Thanks for your suggestions, after testing some scenarios I decided to merge with the standalone one as it's used only to tests propagation and sampling priority.